### ISSUE TITLE
https-everywhere does not work on firefox nightly anymore

### ISSUE THREAD

**Issue by unknownmhack** [EFForg/https-everywhere#11822]
because Firefox is changing to web-extensions, the extension is blocked on Firefox nightly

**Comment by cschanaj** [comment]
https://github.com/EFForg/https-everywhere/issues/11759#issuecomment-321386955

**Comment by jeremyn** [comment]
Closing as a duplicate of #11759 .

**Comment by koops76** [comment]
@unknownmhack https://www.eff.org/files/https-everywhere-test/index.html
Download the latest WebExtensions-only version here.
